desktop/components/clock/view.coffee

```diff
@@ -42,12 +44,18 @@ module.exports = class ClockView extends Backbone.View
         @$('.clock-header').html "Opening in"
         @toDate = @model.get 'offsetStartAtMoment'
       when 'open'
-        @$('.clock-header').html "#{@modelName} closes in:"
+        if isAuction
```

Comment:
#minor but I just noticed that this `@modelName` was intended to be configurable, so I wonder if we could pass in `'Auction'` or `'Sale'` correctly when this [component is instantiated](https://github.com/artsy/force/blob/master/desktop/components/clock/react.js#L14) and avoid this whole back-and-forth.